components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,741 @@
+
+We refer to these as the discrete equations, but time derivative remains continuous. The time discretization is described in the [time stepping design document](TimeStepping.md).
+
+### Momentum Dissipation
+
+The discretized momentum dissipation ${ \bf D}^u_{e,k}$ may include these terms, which are detailed in the subsections below.
+
+$$
+{ \bf D}^u_{e,k} =  \nu_2 \nabla^2 u_{e,k} - \nu_4 \nabla^4 u_{e,k} +
+\frac{\partial }{\partial z} \left( \nu_v \frac{\partial u_{e,k}}{\partial z} \right)
+$$ (discrete-mom-del2)
+
+#### Laplacian dissipation (del2)
+
+$$
+ \nu_2 \nabla^2 u_{e,k} = \nu_2 \left( \nabla D_{i,k} - \nabla^{\perp} \zeta_{v,k} \right)
+$$ (discrete-mom-del2)
+
+where $D$ is divergence and $\zeta$ is relative vorticity. See [Omega V0 Section 3.3.4](OmegaV0ShallowWater.md#del2-momentum-dissipation)
+
+#### Biharmonic dissipation (del4)
+As in [Omega V0 Section 3.3.5](OmegaV0ShallowWater.md#del4-momentum-dissipation), biharmonic momentum dissipation is computed with two applications of the Del2 operator above.
+
+$$
+ - \nu_4 \nabla^4 u_{e,k}
+= - \nu_4 \nabla^2 \left( \nabla^2 u_{e,k} \right)
+$$ (discrete-mom-del4)
+
+#### Vertical momentum diffusion
+Vertical derivatives may be computed with either $z$ or $p$ as the independent variable,
+
+$$
+\frac{\partial }{\partial z} \left( \nu_v \frac{\partial u}{\partial z} \right)
+= \frac{\partial }{\partial p}\frac{\partial p}{\partial z} \left( \nu_v \frac{\partial u}{\partial p} \frac{\partial p}{\partial z}\right)
+= \rho g^2\frac{\partial }{\partial p} \left( \nu_v \rho \frac{\partial u}{\partial p} \right).
+$$ (mom-vert-diff-z-p)
+
+We choose to use $z$ values for simplicity. A single vertical derivative of an arbitrary variable $\varphi$ at mid-layer is
+
+$$
+\frac{\partial \varphi_k}{\partial z}
+= \frac{\varphi_k^{top} - \varphi_k^{bot} }{z_k^{top} - z_k^{bot}}
+$$ (vertderiv1)
+
+and a second derivative is
+
+$$
+\frac{\partial }{\partial z} \left(
+\frac{\partial \varphi_k}{\partial z} \right)
+=
+\frac{1}{z_{k}^{top} - z_{k+1}^{top}} \left(
+\frac{\varphi_{k-1} - \varphi_k }{z_{k-1}^{mid} - z_k^{mid}}
+ -
+\frac{\varphi_{k} - \varphi_{k+1} }{z_{k}^{mid} - z_{k+1}^{mid}}
+\right)
+$$ (vertderiv2)
+
+Thus, the vertical momentum diffusion is
+
+$$
+\frac{\partial }{\partial z} \left( \nu_v \frac{\partial u_{e,k}}{\partial z} \right)
+=
+\frac{1}{z_{e,k}^{top} - z_{e,k+1}^{top}} \left(
+\nu_{e,k}^{top}
+\frac{u_{e,k-1} - u_k }{z_{e,k-1}^{mid} - z_k^{mid}}
+ -
+\nu_{e,k+1}^{top}
+\frac{u_{e,k} - u_{e,k+1} }{z_{e,k}^{mid} - z_{e,k+1}^{mid}}
+\right)
+$$ (discrete-mom-vert-diff)
+
+The native locations for $z$ and $\nu$ are at the mid-layer and cell center, so they must be interpolated to the vertical layer interface and horizontal edge for this computation. This stencil is applied as an implicit tri-diagonal solve at the end of the time step. See details in the [tridiagonal solver design document](TridiagonalSolver) and forthcoming vertical mixing design document.
```

Comment:
\nu usually lives on the vertical interfaces, not mid layer.

Reply:
Thanks. I realized this sentence is not necessary, as we compute $\nu$ wherever it is needed.